docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -0,0 +1,178 @@
+```
+
+Gateway API and OSSM have been installed successfully and are ready to use. We are now ready to create Gateways and HTTPRoutes.
+
+## Gateway API Example
+
+Let’s now use Gateway API to create a Gateway and Route. In this example, we will convert a simple HTTP OpenShift Route into roughly equivalent Gateway API objects (Gateway, HTTPRoute). This is what the OpenShift Route example would look like:
+```yaml
+apiVersion: v1
+kind: Route
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  host: demo.gwapi.${DOMAIN}
+  to:
+    kind: Service
+    name: foo-app
+  port:
+    targetPort: 8080
+```
+
+First, create the Gateway with a Listener that will listen for requests on a URL under the cluster domain. The Gateway must be created in the openshift-ingress namespace and specify the GatewayClass created earlier. By default, Istio will automatically provision a gateway deployment and service with the same name upon creation of this Gateway object:
+```yaml
+$ DOMAIN=$(oc get ingresses.config/cluster -o jsonpath={.spec.domain})
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+  namespace: openshift-ingress
+spec:
+  gatewayClassName: openshift-default
+  listeners:
+  - name: demo
+    hostname: "*.gwapi.${DOMAIN}"
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+```
+
+When the Gateway from the previous step is created, the ingress operator will automatically create a DNSRecord with the hostname from the listener:
+```shell
+$ oc get dnsrecord -n openshift-ingress
+NAME                         AGE
+gateway-7b49f58c6-wildcard   6m11s
+```
+
+Next, let’s create and configure a namespace called demo-app that will have our HTTPRoute object and our example application:
+```shell
+$ oc create namespace demo-app
+```
+
+Create a demo application deployment and service using oc new-app, our cakephp-ingress-demo project, and the foo git branch. This will create a simple server, foo-app, that will allow us to test connectivity. Then use the oc rollout command to wait for it to be rolled out:
+```shell
+$ oc new-app -n demo-app --name foo-app https://github.com/openshiftdemos/cakephp-ingress-demo#foo
+$ oc rollout status deployment -w -n demo-app foo-app
+```
+
+Create the HTTPRoute that will direct requests for http.example.com to our new backend server, foo-app. Note that the parentRefs points to the Gateway, rules specify a backendRefs to our service, and hostnames align with the Gateway’s Listeners hostnames:
+```yaml
+$ oc apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http
+  namespace: demo-app
+spec:
+  parentRefs:
+  - name: gateway
+    namespace: openshift-ingress
+  hostnames: ["demo.gwapi.${DOMAIN}"]
+  rules:
+  - backendRefs:
+    - name: foo-app
+      port: 8080
+EOF
+```
+Wait for the gateway deployment (Envoy) to be ready:
```

Comment:
I wouldn't even mention Envoy here.  If you do, it should be introduced much earlier, and not with just one word :)